### PR TITLE
Write blog post about natural language linting

### PR DESCRIPTION
**Pull request by TimKam**
Feedback welcome :-)

### CODE REVIEW

File: _posts/17-05-2017-natural-language-linting.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  - being
+  - been
+  - am
+  - is
+  - isn't
+  - are
+  ...
+```
+
+Use cases for custom rules are, for example:
```
Comment (hilton) [inline _posts/17-05-2017-natural-language-linting.md]: _You can use custom rules to: … ensure…_

File: _posts/17-05-2017-natural-language-linting.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+* ensuring the consistent use of language (`click the button` vs `click on the button`).
+
+### Automated testing
+Most linters can run on any popular desktop or server operating system and provide machine-readable output.
+This makes them great tools for automated testing.
+Setting up a system that continuously checks your [Markdown](https://daringfireball.net/projects/markdown/)-based website content against your configured linter rules requires little more than basic technical literacy.
+Still, a detailed explanation on how to do this might be worth its own blog post ;-)
+
+## Technology overview
+While natural language linters are still a niche phenomena, a couple of them gained some traction during the last years.
```
Comment (hilton) [inline _posts/17-05-2017-natural-language-linting.md]: _phenomenon_ (singular)

File: _posts/17-05-2017-natural-language-linting.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+Use cases for custom rules are, for example:
+
+* ensuring you removed all occurrences of outdated product names,
+* ensuring domain-specific terms are spelled correctly (it’s `JavaScript` and not `Java Script`),
+* ensuring the consistent use of language (`click the button` vs `click on the button`).
+
+### Automated testing
+Most linters can run on any popular desktop or server operating system and provide machine-readable output.
+This makes them great tools for automated testing.
+Setting up a system that continuously checks your [Markdown](https://daringfireball.net/projects/markdown/)-based website content against your configured linter rules requires little more than basic technical literacy.
```
Comment (hilton) [inline _posts/17-05-2017-natural-language-linting.md]: That's pretty close to saying that _it’s easy_ when you actually need to know which a lot of separate thing to set that up. I’d change _requires little more than…_ to _only requires standard test automation skills._ or something.

**Comment by TimKam** [comment]
@hilton Feel free to improve the intro, I'm not entirely happy with it.

**Pull request merged**